browser.js

```diff
@@ -0,0 +1,11 @@
+var EventEmitter = require('events').EventEmitter
+var getBrowserRTC = require('get-browser-rtc')
+
+module.exports = function (opts) {
+  var wrtc = new EventEmitter()
+  // If electron-webrtc is required in the browser, return the browser RTC implementation.
+  return Object.assign(wrtc, getBrowserRTC(), {
```

Comment:
Maybe we should get the `assign` function from the `object-assign` package? Unfortunately some browsers don't support `Object.assign`.